pkg/apis/kops/componentconfig.go

```diff
@@ -159,7 +159,14 @@ type KubeletConfigSpec struct {
 type KubeProxyConfig struct {
 	Image string `json:"image,omitempty"`
 	// TODO: Better type ?
-	CPURequest string `json:"cpuRequest,omitempty"` // e.g. "20m"
+	// CPURequest, cpu request compute resource for kube proxy e.g. "20m"
```

Comment:
Why don't we use ` v1.ResourceRequirements`??  We have this already modeled, why are we using strings to model API values we already have.

Reply:
@chrislovecnm CPURequest was already a string, so we just extended that pattern for the other three in order to not break backward compatibility.

Reply:
😞 well that is messy.  

@justinsb options?  I would like to clean this up a bit.  We could support both.  Have a struct using `ResourceRequirements` and leaving CPURequest string as well.

Reply:
I'd say we don't have to address in this PR, and if we choose to fix later having 4 fields vs 1 doesn't really change things.